packages/basti-cdk/src/basti-instance.ts

```diff
@@ -155,7 +164,13 @@ export class BastiInstance extends Construct implements IBastiInstance {
       userData: aws_ec2.UserData.custom(BASTION_INSTANCE_CLOUD_INIT),
       vpc: props.vpc,
       vpcSubnets: props.vpcSubnets ?? defaultSubnetSelection,
+      propagateTagsToVolumeOnCreation: true,
     });
+    Tags.of(this.instance).add(BASTION_INSTANCE_ID_TAG_NAME, this.bastiId);
```

Comment:
I made tagging as similar as possible to the current Basti CLI behavior in `main`